### PR TITLE
Include both organization and facility names in test results

### LINKED ISSUE #2883: Test results org/facility name
### Description
When emailing the test results, we don't mention the `org name`, we only mention the `facility name` ("library" in the email)
However, a patient views the test results, we never show the `facility name` we only show the `org name` in the patient experience UI ("UT Bedford " in the ui)

### Additional context
Email:
image

UI:
image

### PR DESCRIPTION
**Pull request by nathancrtr**
## Related Issue or Background Info

* Closes #2883 

## Changes Proposed
- Add `facility_name` to the SendGrid test result email template
- Display facility name in header for test result patient view

## Additional Information
- Adds `facilityName` property to the patient's `lastTest` stored in Redux state

## Screenshots / Demos
**Facility name in patient app header**
https://user-images.githubusercontent.com/27730981/148280862-27c87d09-653c-42ce-8a15-361b9e666235.mov

**Facility name in test result email body**
- **TODO** - will need to activate template & deploy to pentest

### UI
- [x] Any changes to the UI/UX are approved by design 
- [x] Any new or updated content (e.g. error messages) are approved by design 

### Testing
- [x] Includes a summary of what a code reviewer should verify

### Changes are Backwards Compatible
- [x] Database changes are submitted as a separate PR
  - [x] Any new tables that do not contain PII are accompanied by a GRANT SELECT to the no-PHI user
  - [x] Any changes to tables that have custom no-PHI views are accompanied by changes to those views
        (including re-granting permission to the no-PHI user if need be)
  - [x] Liquibase rollback has been tested locally using `./gradlew liquibaseRollbackSQL` or `liquibaseRollback`
  - [x] Each new changeset has a corresponding [tag](https://docs.liquibase.com/change-types/community/tag-database.html)
- [x] GraphQL schema changes are backward compatible with older version of the front-end

### Security
- [x] Changes with security implications have been approved by a security engineer (changes to  authentication, encryption, handling of PII, etc.)
- [x] Any dependencies introduced have been vetted and discussed

## Cloud
- [x] DevOps team has been notified if PR requires ops support
- [x] If there are changes that cannot be tested locally, this has been deployed to our Azure `test`, `dev`, or `pentest` environment for verification

### CODE REVIEW

**Review by zdeveloper — APPROVED** [review]
LGTM! please demo to @tim-skylight

**Review by nickclyde — APPROVED** [review]
Looks great!

File: backend/src/main/java/gov/cdc/usds/simplereport/service/TestResultsDeliveryService.java [hunk 1]
```diff
@@ -45,6 +45,7 @@ public boolean emailTestResults(PatientLink patientLink) {
     Map<String, Object> templateVariables =
         Map.of(
             "facility_name", patientLink.getTestOrder().getFacility().getFacilityName(),
+            "organization_name", patientLink.getTestOrder().getOrganization().getOrganizationName(),
```
Comment (emmastephenson) [inline backend/src/main/java/gov/cdc/usds/simplereport/service/TestResultsDeliveryService.java:48]: Does this require any update to our SendGrid template?

**Review by emmastephenson — APPROVED** [review]
LGTM! Just one question about the emailed results.

**Review by alismx — APPROVED** [review]

File: backend/src/main/java/gov/cdc/usds/simplereport/service/TestResultsDeliveryService.java (hunk 1 above)
Comment (zdeveloper) [reply backend/src/main/java/gov/cdc/usds/simplereport/service/TestResultsDeliveryService.java:48]: we created a v2, will enable the new version after this goes out to prod

**Pull request merged**